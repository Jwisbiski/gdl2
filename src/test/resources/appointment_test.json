{
  "resourceType": "Appointment",
  "status": "proposed",
  "appointmentType": {
    "coding": [
      {
        "system": "http://hl7.org/fhir/v2/0276",
        "code": "FOLLOWUP",
        "display": "FOLLOWUP"
      }
    ]
  },
  "description": "Follow up to check the results of Statin Treatment",
  "participant": [
    {
      "actor": {
        "reference": "Patient/123456"
      },
      "required": "required",
      "status": "needs-action"
    },
    {
      "type": [
        {
          "coding": [
            {
              "system": "http://hl7.org/fhir/v3/ParticipationType",
              "code": "ATND"
            }
          ]
        }
      ],
      "required": "required",
      "status": "needs-action"
    }
  ],
  "requestedPeriod": [
    {
      "start": "{$gt2000}",
      "end": "{$gt2001}"
    }
  ]
}