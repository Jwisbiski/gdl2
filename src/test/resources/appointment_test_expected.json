{
  "resourceType": "Appointment",
  "status": "proposed",
  "appointmentType": {
    "coding": [
      {
        "system": "http://hl7.org/fhir/v2/0276",
        "code": "FOLLOWUP",
        "display": "FOLLOWUP"
      }
    ]
  },
  "description": "Follow up to check the results of Statin Treatment",
  "participant": [
    {
      "actor": {
        "reference": "Patient/123456"
      },
      "required": "required",
      "status": "needs-action"
    },
    {
      "type": [
        {
          "coding": [
            {
              "system": "http://hl7.org/fhir/v3/ParticipationType",
              "code": "ATND"
            }
          ]
        }
      ],
      "required": "required",
      "status": "needs-action"
    }
  ],
  "requestedPeriod": [
    {
      "start": "2013-04-20",
      "end": "2013-04-25"
    }
  ]
}